erpnext/accounts/doctype/purchase_invoice/purchase_invoice_list.js

```diff
@@ -23,5 +23,42 @@ frappe.listview_settings['Purchase Invoice'] = {
 		} else if(flt(doc.outstanding_amount)==0 && doc.docstatus==1) {
 			return [__("Paid"), "green", "outstanding_amount,=,0"];
 		}
+	},
+	onload: function (listview) {
+		const action = () => {
+            const selected_docs = listview.get_checked_items();
```

Comment:
Change the indentation for the new changes to tabs

Reply:
crap